apache_exporter.go

```diff
@@ -307,7 +378,7 @@ func main() {
 
 	// Parse flags
 	log.AddFlags(kingpin.CommandLine)
-	kingpin.Version(version.Print("apache_exporter"))
+	kingpin.Version(Version)
```

Comment:
Why should the apache version be printed here?

If you run `./exporter --version` I would expect to see exporter version

Reply:
```Verison``` here is not apache version, I think it can be removed, I was not really familiar with prometheus.version package. It's the exporter version

Instead apache version is object of a new metrics ```apache_version```